packages/driver/src/dom/jquery.js

```diff
@@ -1,6 +1,17 @@
 const $ = require('jquery')
 const _ = require('lodash')
 
+// Check if element contains a child element
+const contains = (doc, obj) => {
+  if (!doc || !obj) {
+    return false
+  }
+
+  return $.contains(doc, obj) ||
+      $.contains(doc, obj.getRootNode().host) ||
```

Comment:
is this enough? what if we have:

```html
#document <-- THIS IS "doc"
  <my-element-1>
    #shadow-root
    <my-element-2>
      #shadow-root
        <div> <-- THIS IS "obj"
```

`contains(document, divEl.getRootNode().host)` would result in `contains(document, myElement2)` which isn't directly true